packages/core/docs/commercetools/authorization-strategy.md

```diff
@@ -0,0 +1,2 @@
+# Authorization strategy
```

Comment:
```suggestion
# Authorization
```
and perhaps it's worth at least informing how to override the token and in which cookie we're storing the data?